tests/manage/test_pvc_disruptive.py

```diff
@@ -21,43 +61,171 @@ class BaseDisruption(ManageTest):
     pvc_obj = None
     storage_type = None
 
+    def create_pvcs(self):
+        """
+        Create PVCs
+
+        Returns:
+            list: List of PVC objects
+        """
+        # Parameters for PVC yaml as dict
+        pvc_data = load_yaml_to_dict(constants.CSI_PVC_YAML)
+        pvc_data['metadata']['namespace'] = self.namespace
+        pvc_data['spec']['storageClassName'] = self.sc_obj.name
+        pvc_data['metadata']['name'] = helpers.create_unique_resource_name('test', 'pvc')
+
+        # Create 5 PVCs
+        pvc_objs = create_multiple_pvc(5, pvc_data)
+        return pvc_objs
+
+    def create_pods(self, pvc_objs_list):
```

Comment:
move to pod.py

Reply:
Added as helper function because 'create_pod' and 'wait_for_resource_state' are already available in helpers.py which can be used in 'create_pods'.
Can be moved to pod.py If it is appropriate to use tests/helpers module in pod.py